7a_temp_coop_munge.yml

```diff
@@ -46,7 +46,7 @@ targets:
   7a_temp_coop_munge/log/7a_temp_coop_munge_tasks.ind:
     command: loop_tasks(task_plan = coop_munge_taskplan,
       task_makefile = '7a_temp_coop_munge_tasks.yml',
-      num_tries = 3)
+      num_tries = 1)
```

Comment:
I think we may want to leave `num_tries = 3` unless there was a specific reason we dropped it lower?

Reply:
@jread-usgs and I changed this value from 1 to 3 while troubleshooting so that we could save some wait time. We decided to commit the change because `num_tries` is typically intended to query a flaky web service multiple times without breaking the pipeline. In the case of this specific pipeline, it's just repeating the error 3 times - if it fails once it will definitely fail thrice.

Reply:
I suggested 1 because this isn't a brittle function call that fails sometimes and works other times (like a flakey web request). My understanding is if any one of these fails once, they will fail the other two times, hence no need for a retry here.